krux/parser.py

```diff
@@ -244,7 +251,7 @@ def __init__(self, env_var_prefix=None, *args, **kwargs):
         :type kwargs: dict
         """
         # Call to the superclass to bootstrap.
-        super(KruxGroup, self).__init__(*args, **kwargs)
+        super(KruxGroup, self).__init__(**kwargs)
```

Comment:
This calls `argparse._ArgumentGroup.__init__(self, container, title=None, description=None, **kwargs))`, but throws away the positional argument `container`, I think?

Reply:
`container` is included in `kwargs`. I only pull out `env_var_prefix` so everything else is in `kwargs`.

Reply:
Also changed it so that you cannot call this function with positional argument to prevent the more of these bugs. Since the only place it is being called is in `KruxParser` (for now), this should be safe.